osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/README.md

```diff
@@ -38,55 +40,46 @@ Tail the logs:
 docker logs -f CONTAINER
 ```
 
-## Add MQTT RTU Device in the DB
```

Comment:
Include this section in the README.md. In case somebody wants to add a device using these statements instead of the web-service.